src/components/EmailInviteDialog.js

```diff
@@ -267,7 +295,12 @@ const EmailInviteFriendsDialog = ({
           ) : (
             <Fade in>
               <div className={classes.sentBox}>
-                <img src={catsSent} height="120px" width="240px" alt="cats2" />
+                <img
+                  src={afterShareImage}
+                  height="120px"
+                  width="240px"
+                  alt={`${alt}2`}
+                />
```

Comment:
on the Button on line 311, you'll want to remove the color from classes.sendButton and you'll want to set the color to "secondary".  It should show teal on both cats and seas